tools/ci_connector_ops/ci_connector_ops/qa_engine/enrichments.py

```diff
@@ -33,10 +38,13 @@ def get_enriched_catalog(oss_catalog: pd.DataFrame, cloud_catalog: pd.DataFrame)
     enriched_catalog.columns = enriched_catalog.columns.str.replace(
         "(?<=[a-z])(?=[A-Z])", "_", regex=True
     ).str.lower() # column names to snake case
```

Comment:
Nit: might be nice to move this into a function instead of needing comment to explain what its doing:
```
enriched_catalog.columns = enriched_catalog.columns.to_snake_case()
```